### PR TITLE
Issue 924: Provide a json file for Marathon app groups to run Pulsar on DCOS

### PR DESCRIPTION
**Pull request by zhaijack**
### Motivation

There was a requirement to run Pulsar on DCOS in #917 , but Universe Package is not support well now. this is to create a json file for Marathon app groups to run Pulsar on DCOS.

### Modifications

add a json file to deploy pulsar as a Marathon group on DCOS.

### Result

could use command `dcos marathon group add PulsarGroups.json` to deploy Pulsar on DCOS.

### CODE REVIEW

**Comment by jiazhai** [comment]
@sijie.Iit just need one cli command to do the deploy `dcos marathon group add PulsarGroups.json
`, adding a page to make this more clear for this deploy.

**Comment by zhaijack** [comment]
@sijie. It just need one cli command to do the deploy dcos marathon group add PulsarGroups.json, adding a page to make this more clear for this deploy.

**Comment by zhaijack** [comment]
seems picture on website not auto fill the browser window, opened issue #951 tracking the picture show issue.

**Comment by lucperkins** [comment]
@zhaijack The content of the doc looks great! I think it could use some reworking from a native speaker, but I'll do that in a separate PR. Ship it!

**Comment by merlimat** [comment]
retest this please

File: deploy/dcos/PulsarGroups.json [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+          "id": "/pulsar/monitor/prom",
+          "cpus": 0.5,
+          "mem": 512,
+          "instances": 1,
+          "constraints": [["hostname", "UNIQUE"]],
+          "container": {
+            "type": "DOCKER",
+
+            "docker": {
+              "image": "zhaijia/prometheus",
```
Comment (rdhabalia) [inline deploy/dcos/PulsarGroups.json]: is there any thing additional present into docker image compare to `https://hub.docker.com/r/prom/prometheus/` ? If not then should we replace it with `prom/prometheus/`?

Comment (jiazhai) [reply deploy/dcos/PulsarGroups.json]: Yes, the configuration file is special for DC/OS

Comment (rdhabalia) [reply deploy/dcos/PulsarGroups.json]: then should we we add docker file in this repo and push image under `apachepulsar`?

Comment (sijie) [reply deploy/dcos/PulsarGroups.json]: +1 to @rdhabalia suggested.

**Pull request merged**